### PR TITLE
add README.md to katello-client-bootstrap RPM

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: katello-client-bootstrap/katello-client-bootstrap.spec [hunk 1]
```diff
@@ -43,6 +43,7 @@ mkdir -p %{buildroot}/%{_var}/www/html/pub/
 cp bootstrap.py %{buildroot}%{_var}/www/html/pub/bootstrap.py
 
 %files
+%doc README.md
```
Comment (ehelms) [inline katello-client-bootstrap/katello-client-bootstrap.spec:46]: Should this go in a -doc sub-package?

Comment (evgeni) [reply katello-client-bootstrap/katello-client-bootstrap.spec:46]: for a 19KB big README? IMHO not.

**Comment by evgeni** [comment]
[test]

**Comment by jlsherrill** [comment]
ACK thanks @evgeni !

**Pull request merged**